mubench.reviewsite/templates/detector.phtml

```diff
@@ -231,6 +232,15 @@ $markdown_parser = new Parsedown();
                 <td class="<?= $style_classes ?>" style="max-width: 300px;">
                     <?= $misuse->hasReviewed($user) ? $misuse->getReview($user)->getComment() : "" ?>
                 </td>
+                <td class="<?= $style_classes ?>">
+                    <? if($misuse->hasSufficientReviews()) : ?>
```

Comment:
There needs to be an agreement/resolution. See my reply to your respective question in #88.